### PR TITLE
Misc low hanging fruit for 1.12.2

### PR DESCRIPTION
**Pull request by thdaele**

### CODE REVIEW

**Comment by SpaceWalkerRS** [comment]
there seems to be a class name conflict in 1.12

```
non-unique class target name mappings:
  [net/minecraft/unmapped/C_6260518$C_3395697, net/minecraft/unmapped/C_6260518$C_8259883] -> net/minecraft/client/gui/screen/inventory/menu/CreativeInventoryScreen$CreativeInventorySlot
```

**Comment by thdaele** [comment]
Sorry I will look at it later today, these are mappings that I still had lying around from some time ago

File: mappings/1.12.2#17w43a.tinydiff [hunk 1]
```diff
@@ -3043,9 +3043,9 @@ c	net/minecraft/unmapped/C_3020744
 		p	4		duration	
 	m	(Lnet/minecraft/unmapped/C_2348249;)V	m_7971793	buttonClicked	
 		p	1		button	
-	m	(II)V	m_0986658	m_0986658	
-		p	1		p_1	
-		p	2		p_2	
+	m	(II)V	m_0986658	setWidthHeight
```
Comment (SpaceWalkerRS) [inline mappings/1.12.2#17w43a.tinydiff:3046]: `setDimensions` or `setSize` would be a bit more concise

File: mappings/1.3-pre-07261249#12w30e-server.tinydiff [hunk 2]
```diff
@@ -6046,7 +6046,7 @@ c	net/minecraft/unmapped/C_3930157
 	m	()Ljava/lang/String;	m_8387532	m_8387532	
 c	net/minecraft/unmapped/C_3940616	net/minecraft/client/render/block/entity/BlockEntityRenderDispatcher	
 	f	Ljava/util/Map;	f_5037161	renderers	
-	f	Lnet/minecraft/unmapped/C_3940616;	f_6470555	INSTANCE	
+	f	Lnet/minecraft/unmapped/C_3940616;	f_6470555	getWorld
```
Comment (SpaceWalkerRS) [inline mappings/1.3-pre-07261249#12w30e-server.tinydiff:6049]: is this a mistake? the new name is a method name given to a field

File: mappings/17w13b#17w14a.tinydiff [hunk 3]
```diff
@@ -714,13 +714,13 @@ c	net/minecraft/unmapped/C_9590849
 	m	(Lnet/minecraft/unmapped/C_0539808;)Ljava/util/Collection;	m_7892993	m_7892993	
 		p	1		p_1	
 c	net/minecraft/unmapped/C_9639106
-	m	(II)V	m_1779891		m_1779891
-		p	1			p_1
-		p	2			p_2
+	m	(II)V	m_1779891		conditionalRenderToolTip
```
Comment (SpaceWalkerRS) [inline mappings/17w13b#17w14a.tinydiff:717]: is a `renderTooltipIf<condition>` feasible here?

File: mappings/17w14a#17w15a.tinydiff [hunk 4]
```diff
@@ -714,8 +714,8 @@ c	net/minecraft/unmapped/C_8124234
 		p	3			p_3
 		p	4			p_4
 c	net/minecraft/unmapped/C_8166793
-	m	()V	m_0508917		m_0508917
-	m	()V	m_1012213		m_1012213
+	m	()V	m_0508917		createSubScreens
```
Comment (SpaceWalkerRS) [inline mappings/17w14a#17w15a.tinydiff:717]: subscreen sounds a little awkward, are these not like tabs?

**Review by SpaceWalkerRS — CHANGES_REQUESTED** [review]

File: mappings/1.12.2#17w43a.tinydiff (hunk 1 above)
Comment (thdaele) [reply mappings/1.12.2#17w43a.tinydiff:3046]: Yeah setSize seems better here

File: mappings/1.3-pre-07261249#12w30e-server.tinydiff (hunk 2 above)
Comment (thdaele) [reply mappings/1.3-pre-07261249#12w30e-server.tinydiff:6049]: Yeah seems that I somehow made a mistake here

File: mappings/17w13b#17w14a.tinydiff (hunk 3 above)
Comment (thdaele) [reply mappings/17w13b#17w14a.tinydiff:717]: It just checks if you are not dragging a stack and that there is a stack in the slot you are hovering.
So there are no additional conditions than you expect from renderToolTip.
Iirc I named it like that bc I thought I couldn't use renderToolTip bc that method name is already used (but it is with diff args so I can actually use it or at least enigma allows it)

File: mappings/17w14a#17w15a.tinydiff (hunk 4 above)
Comment (thdaele) [reply mappings/17w14a#17w15a.tinydiff:717]: Yeah that name also didn't feel 100% right for me, tabScreen indeed better

**Comment by SpaceWalkerRS** [comment]
Hmm, now it's `C_0539808`/`getClientPermissionLevel` that is causing conflicts - it is called `getPermissions` elsewhere in 1.13.

Was `getPermissions` already taken when you mapped it? Cause then we have a match issue on our hand.

**Comment by thdaele** [comment]
It showed up in enigma as unmapped, I can map it to getPermissions without a problem tho

**Comment by SpaceWalkerRS** [comment]
Alright, then just rename it to `getPermissions` for now to solve the conflict, then we can look into whether the name makes sense in all versions later.

**Review by SpaceWalkerRS — APPROVED** [review]
excellent! the workflow succeeds now, thanks for the mappings!

**Pull request merged**